Plugins/Public/playercntl_plugin/Main.cpp

```diff
@@ -915,13 +915,16 @@ USERCMD UserCmds[] =
 	{ L"/set movecharcode",	Rename::UserCmd_SetMoveCharCode,	L"Usage: /set movecharcode <code>" },
 	{ L"/restart",		Restart::UserCmd_Restart,		L"Usage: /restart <faction>" },
 	{ L"/showrestarts",	Restart::UserCmd_ShowRestarts,	L"Usage: /showrestarts" },
-	{ L"/givecash",		GiveCash::UserCmd_GiveCash,		L"Usage: /givecash <charname> <cash> [anon]" },
-	{ L"/givecasht",	GiveCash::UserCmd_GiveCashTarget,		L"Usage: /givecasht <cash> [anon]" },
-	{ L"/givecasht*",	GiveCash::UserCmd_GiveCashTarget,		L"Usage: /givecasht <cash> [anon]" },
```

Comment:
You appear to have remove /givecasht*

Reply:
None of the others has the * so I didn't think it was needed.